<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_add</class>
 <widget class="QDialog" name="Dialog_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Adding Continent</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Dialog_add {
background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #00B4DB, stop:1 #0083B0);
}</string>
  </property>
  <widget class="QLabel" name="label_activity">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>10</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_activity {
color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Adding Continent</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>221</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="lineEdit_nameCont"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_sqrCont"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_landmassCont"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>230</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="styleSheet">
       <string notr="true">#pushButton_cancel {
width: 100px;
height: 35px;
}</string>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="styleSheet">
       <string notr="true">#pushButton_ok {
width: 100px;
height: 37px;
border-style: solid;
border-width: 2px;
border-color: #ffffff;
color: #ffffff;
background-color: #D93471;
font-weight: bold;
}
#pushButton_ok:hover {
background-color: #ef6799;
border-color: #b8ffbd;
}
</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>75</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_name {
color: #ffffff;
font-weight: bold;
font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_landmass">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>158</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_landmass {
color: #ffffff;
font-weight: bold;
font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Landmass</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_square">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>120</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label_square {
color: #ffffff;
font-weight: bold;
font-size: 20px;
}</string>
   </property>
   <property name="text">
    <string>Square</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
